tools/bioformats2raw/bf2raw.xml

```diff
@@ -0,0 +1,134 @@
+<tool id="bf2raw" name="Convert to OME-Zarr" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@">
+    <description>with Bioformats</description>
+    <edam_operations>
+        <edam_operation>operation_3443</edam_operation>
+    </edam_operations>
+    <macros>
```

Comment:
https://github.com/BMCV/galaxy-image-analysis/actions/runs/6876155993/job/18701399534?pr=72#step:7:383

> Applying linter xml_order... WARNING
.. WARNING: Best practice violation [macros] elements should come before [edam_operations]

Please run `planemo lint` locally before pushing :)

Reply:
You are right. Now fixed.